bolt/internal/internal.proto

```diff
@@ -93,6 +93,7 @@ message Cell {
 	repeated int64 yranges  = 8; // Limits of the y-axes
 	repeated string ylabels = 9; // Labels of the y-axes
 	string type             = 10; // Cell visualization type
+	map<string, Axis> axes  = 11; // Axes represent the graphical viewport for a cell's visualizations
```

Comment:
@jaredscheib I was going for the more generic usage, but I'm happy to change it if it's confusing:

> **viewport**: _noun_
>    1. a window in a spacecraft or in the conning tower of an oil rig.
>    2. [computing] a framed area on a display screen for viewing information.